beta-src/src/components/ui/WDButton.tsx

```diff
@@ -6,20 +6,25 @@ interface WDButtonProps {
   color?: "primary" | "secondary";
   disabled?: boolean;
   onClick?: React.MouseEventHandler<HTMLButtonElement> | undefined;
+  startIcon?: React.ReactNode | undefined;
+  variant?: "text" | "contained" | "outlined";
```

Comment:
remove variant...we aren't using this, right? if we are, we should be using the same type for variant as material ui does